app/assets/js/rtMedia.backbone.js

```diff
@@ -2913,3 +2919,13 @@ function rtmedia_disable_popup_navigation( $selector ){
 		} );
 	} );
 }
+
+/** 
+ * Function that smooth scrolls to the latest comment in rtMedia.
+ *  Created on 23-Nov-2020 by Vipin Kumar Dinkar <vipin.dinkar@rtcamp.com>
+ */
+const rtMediaScrollComments = () => {
+	const commentBox = document.getElementById('rtmedia_comment_ul');
+	const commentsToScroll = (document.getElementById('rtmedia_comment_ul').offsetHeight)*1000; 
+	commentBox.scrollTo({top: commentsToScroll, behavior: 'smooth'});
+}
```

Comment:
@NicestRudeGuy  

For better readability can you please add proper spacing?

it should be like this:

```
        const commentBox         = document.getElementById( 'rtmedia_comment_ul' );
	const commentsToScroll = ( document.getElementById( 'rtmedia_comment_ul' ).offsetHeight ) * 1000; 
	commentBox.scrollTo( { top: commentsToScroll, behavior: 'smooth' } ); 
```

Reply:
sure

Reply:
@NicestRudeGuy  This has not been changed yet.